### ISSUE TITLE
Unhandled Promise Rejection Warning

### ISSUE THREAD

**Issue by redaxmedia** [hilleer/kill-port-process#17]
Please fix your promises by adding a catch() block, killing a port that does not exists will cause following under latest nodejs.

## Error

```
(node:3032) UnhandledPromiseRejectionWarning: Error: Couldn't find a process with port `7000`
    at getPort (...\node_modules\pid-from-port\index.js:29:9)
    at ...\node_modules\pid-from-port\index.js:51:32
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:3032) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)

(node:3032) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by hilleer** [comment]
Hi @redaxmedia ,

Thanks for opening this issue. 

Can you please include some info on what OS you are using and what node version?

**Comment by hilleer** [comment]
I have published a version `2.3.0` that should solve your issue. 

However, if it does not, please supply the requested information.
